### PR TITLE
add cmd color themes

### PR DESCRIPTION
**Pull request by freignat91**
Related to #477 

Have amp commands color themes to adapt colors to different terminals L&F.
Default is the "magenta" theme created by Neha, but it doesn't look good on Ubuntu  default terminal which are magenta dark by default.
Added one named "dark" to be back to the original colors (white in place of magenta)

to change it, update $HOME/.amp.yaml file adding this line for instance:
cmdTheme: dark

For now only 2 themes (default or dark) 

Test:
- make install
- start/stop amp: see the regular lines are magenta
- update .amp.yml as explained
- start/stop amp: see the regular lines are white

It's possible to add new theme updating cmd/amp/platform_manager.go

### CODE REVIEW

File: cmd/amp/platform_manager.go [hunk 1]
```diff
@@ -615,6 +616,51 @@ func (s *ampManager) close() {
...
+				return strings.Trim(param[1], " ")
+			}
+		}
+	}
+	return ""
+}
+
+func (s *ampManager) setColors() {
+	theme := s.getTheme()
+	if theme == "white" {
```
Comment (ndegory) [inline cmd/amp/platform_manager.go]: I think the name of the theme should not be based on one of the color of the theme, but should match the terminal theme on which it displays best.
So here white should be black.

**Review by ndegory — COMMENTED** [review]
good idea to rename the column without the color, as it could change with the theme. The naming of the theme should be different though.

File: cmd/amp/platform_manager.go (hunk 1 above)
Comment (freignat91) [inline cmd/amp/platform_manager.go]: Any idea for this name?

Comment (freignat91) [reply cmd/amp/platform_manager.go]: done, renamed "dark", PR description updated too

File: cmd/amp/platform_manager.go [hunk 2]
```diff
@@ -615,6 +616,51 @@ func (s *ampManager) close() {
 	//s.docker.Close()
 }
 
+func (s *ampManager) getTheme() string {
+	homedir, err := homedir.Dir()
+	if err != nil {
+		return ""
+	}
+	file, err := os.Open(path.Join(homedir, ".amp.yaml"))
```
Comment (generalhenry) [inline cmd/amp/platform_manager.go]: why manually check the config?

Comment (freignat91) [reply cmd/amp/platform_manager.go]: yes, you're right, i updated the code to use the regular amp config

**Review by bquenin — APPROVED** [review]
LGTM

**Comment by generalhenry** [comment]
You can also do 
`amp config cmdTheme dark` to switch to dark
and `amp config cmdTheme default` to switch back

**Review by generalhenry — APPROVED** [review]

**Pull request merged**